portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityInMemoryDataCopier.java

```diff
@@ -42,6 +43,9 @@
   private static final AtomicInteger COPY_ITERATION_COUNTER = new AtomicInteger();
   private static final Logger logger = LoggerFactory.getLogger(PortabilityInMemoryDataCopier.class);
 
+  private static final List<String> fatalErrorRegexes = ImmutableList.of("*fatal*"); // TODO: make configurable
```

Comment:
I think var name should still be capitalized since it's a private static final, if we want to follow google java style guide. like how COPY_ITERATION_COUNTER is cap'd.